spec.html

```diff
@@ -30296,7 +30296,7 @@ <h1>Properties of String Iterator Instances</h1>
                 [[StringIteratorNextIndex]]
               </td>
               <td>
-                The integer index of the next string index to be examined by this iteration.
+                The integer index of the next string element (code unit) to be examined by this iteration.
```

Comment:
I am most certainly wrong, but doesn't `StringIterator` go over code **points**?

Reply:
It's true that the `next()` function returns a code-point-worth of string, but `[[StringIteratorNextIndex]]` remembers its position as an ordinary (code-unit-wise) index into the string.

(If it remembered its position as a *code-point* offset, each call to `next()` would have to re-scan the string to know what to look at next.)

Reply:
Thank you for explanation. I believe #1579 can possibly supersede this change?

Reply:
Indeed it can! (Sorry, I wasn't paying attention.)

In the likely event that it gets merged, I'll close this PR.